docs/buildtools/gradle/help.go

```diff
@@ -0,0 +1,16 @@
+package gradle
+
+const Description = "Run Gradle build."
+
+var Usage = []string{`jfrog gradle <tasks and options> [command options]`}
+
+const Arguments string = `	tasks and options
+		Tasks and options to run with gradle command. For example, -b path/to/build.gradle.`
+
+const EnvVar string = `	JFROG_CLI_EXTRACTORS_REMOTE
```

Comment:
I suggest consulting with @sverdlov93 , to see if following the new "options" command he's working on, the env vars should still be here. The same goes for the other files which include env vars.